packages/playwright-core/src/server/registry/browserFetcher.ts

```diff
@@ -36,7 +36,7 @@ export async function downloadBrowserWithProgressBar(title: string, browserDirec
 
   const zipPath = path.join(os.tmpdir(), downloadFileName);
   try {
-    const retryCount = 3;
+    const retryCount = 5;
     for (let attempt = 1; attempt <= retryCount; ++attempt) {
```

Comment:
Replave `retryCount` with `downloadURLs.length`, otherwise it's hard to follow where 5 comes from, given that the list of CDN urls is in a different file.

Reply:
Soon we have downloadURLs = 2 and we want to keep retryCount of 3 so having them aligned seem to be misleading.